dev/com.ibm.websphere.appserver.features/visibility/public/netty-1.0/io.openliberty.netty-1.0.feature

```diff
@@ -11,6 +11,7 @@ Subsystem-Name: Netty Transport Framework 1.0
   io.openliberty.io.netty
```

Comment:
We need to determine if there is any use at all for this public netty-1.0 feature. If not then we should get an issue opened up so we remove it and don't forget about it. This was originally going to be used as a beta guard but at this point, we've abandoned that idea.